CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - [#153] (https://github.com/HewlettPackard/oneview-golang/issues/153) change in the enclosure.go file for editing  wrong fields
 
 ### Oneview Features supported
```

Comment:
We can maybe change this to `Features Supported` like in python, as deployment plab is a image streamer feature